src/main/java/com/puppycrawl/tools/checkstyle/checks/design/DesignForExtensionCheck.java

```diff
@@ -176,44 +176,76 @@
  * &lt;module name=&quot;DesignForExtension&quot;/&gt;
  * </pre>
  * <p>
- * To configure the check to allow methods which have @Override and @Test annotations
+ * Example:
+ * </p>
+ * <pre>
+ * public class A {
+ *
+ *     &#64;BeforeClass
+ *     public int beforeAllTests() { return 1; } // OK, @BeforeClass is ignored by default
+ *
+ *     &#64;AfterClass
+ *     public int afterAllTests() { return 1; }  // OK, @AfterClass is ignored by default
+ *
+ *     &#64;Before
+ *     public int beforeEachTest() { return 1; }  // OK, @Before is ignored by default
+ *
+ *     &#64;After
+ *     public int afterEachTest() { return 1; }  // OK, @After is ignored by default
+ *
+ *     &#64;Test
+ *     public int testSomething() { return 1; }  // OK, @Test is ignored by default
+ *
+ *     &#64;Deprecated
+ *     public int someOldMethod() { return 1; }  // Violation, @Deprecated is not ignored by default
+ *
+ *     &#47;**
+ *  This method performs some task
+ *     *&#47;
+ *     public int someTask() { return 1; } // OK, default requiredJavadocPhrase is in comment
+ *
+ *     public int someTask2() { return 1; } // Violation
+ * }
+ * </pre>
+ * <p>
+ * To configure the check to allow methods which have @Deprecated and @Test annotations
```

Comment:
Better to keep previous wording
`To configure the check to allow methods which have @Override and @Test annotation only`

Reply:
Please do me a favor, remove `to be designed for extension.` in all places
Comment over method does not make method designed for extension. It just allows skip violation on it.